tests/end2end/conftest.py

```diff
@@ -29,6 +29,8 @@
 import os.path
 import operator
 
+from qutebrowser.browser.webengine.spell import get_installed_languages
```

Comment:
`from qutebrowser.browser.webengine import spell` and use `spell.get_installed_languages()`